frontend/src/metabase/entities/schemas.js

```diff
@@ -50,8 +50,8 @@ export default createEntity({
     },
   },
 
-  reducer: (state = {}, { type, payload }) => {
-    if (type === Questions.actionTypes.CREATE) {
+  reducer: (state = {}, { type, payload, error }) => {
+    if (type === Questions.actionTypes.CREATE && !error) {
```

Comment:
This makes me think about other entity reducers handling this kind of actions. I think most of them don't do the `!error` check. Would it make sense to go through them and add them? WDYT? No meaning this specific PR

Reply:
Sure, I'll do that

Reply:
Thank you!